components/alert/alert.css

```diff
@@ -0,0 +1,52 @@
+.alert {
+    padding: 1.2rem;
+    display: flex;
+    flex-wrap: wrap;
+    width: 20rem;
+    justify-content: center;
+    border-radius: 5px;
+}
+
+.alert-success {
+    border: 2px solid #81c784;
+    color: #5dab63;
+}
```

Comment:
Suggestion: You can store the colors in variables so that they can be reused.

Rest all looks good!